charts/fluent-bit/templates/_helpers.tpl

```diff
@@ -125,3 +125,14 @@ autoscaling/v2beta2
 autoscaling/v2
 {{- end -}}
 {{- end -}}
+
+{{/*
+Create the name of OpenShift SecurityContextConstraints to use
+*/}}
+{{- define "fluent-bit.openShift.securityContextConstraints.name" -}}
+{{- if .Values.openShift.securityContextConstraints.create -}}
+  {{ default (include "fluent-bit.fullname" .) .Values.openShift.securityContextConstraints.name }}
+{{- else if .Values.openShift.securityContextConstraints.existingName -}}
+  {{- printf "%s" .Values.openShift.securityContextConstraints.existingName -}}
+{{- end -}}
+{{- end -}}
```

Comment:
@stevehipwell 
You aren't required of SCCs for each time. SCCs only allows to overcome cluster restrictions for Secruity Context, built-in or set by administrators. So it's possible to run Fluent-Bit without additional SCC access provided, but in the particular cases such as hostPath volume required, or UID/GID ranges doesn't met limitations in the current cluster - it could be useful. 

Because of this I don't see default SCC name requires - some OpenShift clusters will be able to launch Fluent-Bit as-is, but others may not.

Reply:
OK, that makes sense but my other review comments still need addressing.